etc/telemetry/otel-tempo-config.yaml

```diff
@@ -0,0 +1,25 @@
+receivers:
```

Comment:
So far we seem to have the pattern of `compose-*.yaml`. I think we should stick to that.

Reply:
thanks for review, PR updated :+1: